config/sitemap.rb

```diff
@@ -60,11 +66,11 @@
     add "/categories/#{category.slug}/"
   end
 
-  [2017, 2016, 2015, 2014, 2013, 2012, 2011, 2010, 2009, 2008, 2007, 2006, 2005, 2004, 2003, 2002, 2001, 2000, 1997, 1996].each do |year|
+  [1996..Time.zone.today.year].to_a.each do |year|
     add "/#{year}/"
   end
 
-  [Book, Episode, Page, Podcast, Video].each do |model|
+  [Book, Episode, Page, Podcast, Video, Tools].each do |model|
```

Comment:
Have you confirmed that this works? 

I wouldn't expect it to:

```ruby
irb(main):001:0> Tools
(irb):1:in `<main>': uninitialized constant Tools (NameError)
```

Instead, I think what we need here is to list out each tool model (in its singular form):
```ruby
Zine
Journal
Issue
Episode
Poster
Sticker
Logo
```

Note: we have "Music" on `/tools`, but it's just a redirect to the band camp page. So we don't need to include it in sitemap.

Reply:
Done.